updatecli/updatecli.d/nodejs-windows.yml

```diff
@@ -42,27 +42,27 @@ conditions:
 
 targets:
   updateVersion:
-    name: Update the NodeJS version in provisioning environment
+    name: Update the NodeJS version in provisioning environment for windows
     sourceid: lastReleaseVersion
     kind: yaml
     spec:
       file: provisioning/tools-versions.yml
-      key: $.nodejs_version
+      key: $.nodejs_windows_version
     scmid: default
-  updateTestHarness:
+  updateWindowsGossTest:
```

Comment:
FWIW, I would have kept "TestHarness", "Goss" is an implementation detail.

Reply:
agreed